### PR TITLE
more on bidding

### PR DESCRIPTION
**Pull request by oxaudo**
This is my progress for today.

I don't want to write any tests for this just yet, cause I have a feeling that a lot of this stuff is going to change.

I'll try to wire some of those buttons to proper urls tomorrow but otherwise I think we might need somebody from auctions to help us out with testing all this....

Oh and this is missing AuctionTimer. I think it could be added directly into ArtworkSidebar, right @mzikherman ?

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit cac3a40b4977a4fcd9a2c420bd56b1ddbe807018

https://deploy-preview-1689--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
Looking good so far! Feel free to merge/iterate/add tests/etc.

> Oh and this is missing AuctionTimer. I think it could be added directly into ArtworkSidebar

Yup was looking at that this afternoon, sidetracked on adding the edition set sorting, but will go back to it now. That `AuctionTimer` component is smart and has everything it needs to render itself in the different states, so yea should be able to be added easily: `...AuctionTimer_sale` and `<AuctionTimer sale={props.sale} />` basically!

**Comment by oxaudo** [comment]
@mzikherman - added some more stuff here to display the selection of nextMaxBid. I think we might want to merge this. Let's maybe chat after lunch and sprint planning to see what the next steps are here.

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
+  ArtworkSidebarBidActionProps,
+  ArtworkSidebarBidActionState
 > {
+  state: ArtworkSidebarBidActionState = {
+    nextMaxBidCents:
+      get(this.props.artwork, a => a.sale_artwork.increments[0].cents) || null,
+  }
+
+  setMaxBid = (newVal: number) => {
+    this.state.nextMaxBidCents = newVal
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx]: This should be using `setState`.

**Pull request merged**